build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
         classpath 'com.palantir.metricschema:gradle-metric-schema:0.5.5'
         classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.9.8'
         classpath 'gradle.plugin.com.palantir.gradle.docker:gradle-docker:0.22.1'
-        classpath 'com.palantir.baseline:gradle-baseline-java:1.8.2'
+        classpath 'com.palantir.baseline:gradle-baseline-java:3.41.0'
```

Comment:
may want to consider a PR before this to just reformat things to minimize the size of this PR to just the things necessary to be gradle-baseline compliant
```
        classpath 'com.palantir.javaformat:gradle-palantir-java-format:1.0.1'

apply plugin: 'com.palantir.java-format'
```

Reply:
I was just taking a quick stab at this in the evening - you're probably right that this would make it easier to merge (although it looks like you're doing a better job over at https://github.com/palantir/atlasdb/pull/5044 so I'll probably just close this in favour of your PR).